docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+services:
+  app:
+    depends_on:
+    - db
+    build:
+      args:
+        user: www
+        uid: 1000
+      context: ./
+      dockerfile: development/Dockerfile
+    image: laravel-image
+    container_name: laravel-app
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - projectroot:/var/www
+      - ./development/php/local.ini:/usr/local/etc/php/conf.d/local.ini
+      - ~/.composer/auth.json:/home/www/.composer/auth.json
+    networks:
+      - app-network
+
+  db:
+    image: mysql:5.7
+    container_name: laravel-db
+    restart: unless-stopped
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: ${DB_DATABASE}
+      MYSQL_ROOT_PASSWORD: ${DB_PASSWORD}
+      MYSQL_PASSWORD: ${DB_PASSWORD}
+      MYSQL_USER: ${DB_USERNAME}
+      SERVICE_TAGS: dev
+      SERVICE_NAME: mysql
+    volumes:
+      - ./development/mysql/my.cnf:/etc/mysql/my.cnf
+      - ./development/mysql/docker-entrypoint-initdb.d/:/docker-entrypoint-initdb.d/
+    networks:
+      - app-network
+
+  nginx:
+    image: nginx:alpine
+    container_name: laravel-nginx
+    restart: unless-stopped
+    ports:
+      - 8000:80
```

Comment:
I am not a fan of binding to port 80 on the host. I would prefer something like support for `VIRTUAL_HOST` like `dory` or the like.

Reply:
Done